### PR TITLE
upgrade tabler to beta-5

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Opened as wip as buttons styles have changed, i need to do a pass on them.
@cconard96 could you do a pass on dark mode, there a lot of changes for this mode in this beta

### CODE REVIEW

**Comment by cconard96** [comment]
Overall it looks good.

We can remove the danger button hack from the _dark.scss file now. The other items in this file are still valid.
```diff
diff --git a/css/includes/_dark.scss b/css/includes/_dark.scss
index 5575799b43..ffd1f388dc 100644
--- a/css/includes/_dark.scss
+++ b/css/includes/_dark.scss
@@ -127,19 +127,6 @@ input[type="color"] {
    border-color: #6d000a;
 }
 
-.btn-danger,
-.btn-outline-danger,
-.btn-ghost-danger {
-   --tblr-btn-color: #642b2b;
-   --tblr-btn-color-darker: #6d000a;
-
-   background-color: var(--tblr-btn-color);
-
-   &:hover {
-      background-color: var(--tblr-btn-color-darker);
-   }
-}
-
 .accordion-item .accordion-button {
    background-color: var(--dark) !important;
 }
```

I'm not too sure about the button borders in dark mode. They should probably have less contrast especially if they aren't primary buttons.
Screenshot from 2021-12-08 04-32-28

**Comment by orthagh** [comment]
> We can remove the danger button hack from the _dark.scss file now

yes I'm on it

**Comment by orthagh** [comment]
> They should probably have less contrast

Not specific to dark mode, light themes also now have a big contrast on buttons

**Comment by orthagh** [comment]
Ready for merge, i fixed most of issues.

**Review by cconard96 — APPROVED** [review]

File: templates/layout/parts/global_search_form.html.twig [hunk 1]
```diff
@@ -31,11 +31,15 @@
...
-         <span class="ti ti-search" aria-hidden="true"></span>
-      </button>
+   <div class="input-group input-group-flat">
+      <input type"text" class="form-control" name="globalsearch" placeholder="{{ __('Search…') }}" />
+      <span class="input-group-text">
+         <a class="link-secondary" title="{{ __('Search…') }}"
+            onclick="$(this).closest('form').submit();"
+            role="button">
+            <span class="ti ti-search" aria-hidden="true"></span>
+         </a>
```
Comment (cedric-anne) [inline templates/layout/parts/global_search_form.html.twig:41]:
```suggestion
         <button type="submit" class="btn btn-link p-0 m-0" title="{{ __('Search…') }}">
            <span class="ti ti-search" aria-hidden="true"></span>
         </button>
```

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**